### DISCUSSION TITLE
how to use limboai in a exist gdextension project

### DISCUSSION THREAD

**Discussion by MAXFROMCODE** [limbonaut/limboai#166] [Q&A]
i have a godot project with gdextension ,i want use limboai in that project ,but i'm dont know how to use it ? also dont know is feasible?

**Comment by limbonaut** [comment]
AFAIK, it's not possible to extend types that are defined in another GDExtension.
You may consider maintaining a fork of LimboAI with your own tasks included as a workaround.
